webapp/src/pages/PerformanceAnalysis.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { screen } from "@testing-library/react";
+import { renderWithRouterAndRedux } from "mocks/utils";
+import PerformanceAnalysis from "./PerformanceAnalysis";
+import { setupServer } from "msw/node";
+import { getDatasetInfoAPIResponse } from "mocks/api/mockDatasetInfoAPI";
+import { PIPELINE_REQUIRED_TIP } from "utils/const";
+
+describe("PerformanceAnalysisComparisonWithoutPipeline", () => {
+  const handlers = [getDatasetInfoAPIResponse];
+  const server = setupServer(...handlers);
+
+  beforeAll(() => server.listen());
+  afterEach(() => server.resetHandlers());
+  afterAll(() => server.close());
+
+  it("should display 'pipeline requied' message if no pipeline selected or exist in the config", () => {
```

Comment:
Where do we test that?

Reply:
Please check my answers [here](https://github.com/ServiceNow/azimuth/pull/420#discussion_r1108578037)

Reply:
Oh, right! Thank you! 🙂 👍 
There is just one little typo:
```suggestion
  it("should display 'pipeline required' message if no pipeline selected or exist in the config", () => {
```

Reply:
oops!. Corrected. done.